design-editor/src/system/assistant/assistant-manager.js

```diff
@@ -52,15 +51,16 @@ class AssistantManager {
     }
 
     /**
-     * Gets script path when script name is tha same as 
+     * Gets script path when script name is tha same as
      * currently opened HTML file. If this JS file doesn't exists
      * it creates new one.
      * @param {DesignEditor} closetDesignEditor - Design Editor instance
      * @returns {string} Path to js file
      * @private
      */
     _getReferenceableScriptPath() {
-        const fileName = pathUtils.getFileName(utils.checkGlobalContext("globalData").fileUrl),
+        const htmlUrl = utils.checkGlobalContext("globalData").fileUrl,
+            fileName = pathUtils.getFileName(htmlUrl),
             name = `${fileName}.js`;
```

Comment:
Looks nice, but as you changed logic this part of the code, you can clean up other variables a little.

Reply:
fixed